deepmedic/logging/tensorboard_logger.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import absolute_import, print_function, division
+
+import tensorflow as tf
+
+
+class TensorboardLogger(object):
+
+    def __init__(self, log_path, tf_graph):
+        self.logger = tf.compat.v1.summary.FileWriter(log_path, tf_graph)
```

Comment:
I would add a note (for us) here that says that this does NOT add a variable to the graph, and thus does not need initialization.

(can be checked by:
```
with tf.variable_scope("others"):
                tensorboard_loggers = self.create_tensorboard_loggers(['train', 'val'], # Main Difference 2.
                                                                      graphTf,
                                                                      create_log=self._params.get_tensorboard_bool())

collection_vars_others = tf.get_collection(tf.GraphKeys.GLOBAL_VARIABLES, scope="others")
print(collection_vars_others)    # Returns empty list []
```